### DISCUSSION TITLE
Feature Request: Playground should offer contextual_tuples as part of assertions

### DISCUSSION THREAD

**Discussion by ghstahl** [openfga/community#20] [Ideas]
I am doing the tutorial: [organization-context-authorization](https://openfga.dev/docs/modeling/organization-context-authorization)  

To check if a user (anne) has access to a feature, I have to burn some golang code.   
```go

body := fgaSdk.CheckRequest{
	TupleKey: &fgaSdk.TupleKey{
		User: fgaSdk.PtrString("anne"),
		Relation: fgaSdk.PtrString("can_view"),
		Object: fgaSdk.PtrString("project:X"),
	},
	ContextualTuples: &fgaSdk.ContextualTuples{
		TupleKeys: []fgaSdk.TupleKey{
			{
				User: fgaSdk.PtrString("anne"),
				Relation: fgaSdk.PtrString("user_in_context"),
				Object: fgaSdk.PtrString("organization:A"),
			}
		}
	}
}
data, response, err := fgaClient.OpenFgaApi.Check(context.Background()).Body(body).Execute()
```

The playground assertions only take in the TupleKey values.  
In short, I would like to test the model fully in the playground.

**Comment by aaguiarz** [comment]
Hi @ghstahl 

Thanks for the request, this makes total sense.
